deploy/helm/sumologic/README.md

```diff
@@ -569,6 +569,7 @@ The following table lists the configurable parameters of the Sumo Logic chart an
 | `metadata.logs.statefulset.extraVolumes`                                         | Additional volumes for logs metadata enrichment (otelcol) pods.                                                                                                                                                                                                                                                                                                      | `Nil`                                                                                                                                                                                |
 | `metadata.logs.statefulset.extraVolumeMounts`                                    | Additional volume mounts for logs metadata enrichment (otelcol) pods.                                                                                                                                                                                                                                                                                                | `Nil`                                                                                                                                                                                |
 | `metadata.logs.statefulset.extraPorts`                                           | Additional exposed ports in logs metadata enrichment (otelcol) pods and service.                                                                                                                                                                                                                                                                                     | `Nil`                                                                                                                                                                                |
+| `metadata.logs.statefulset.extraArgs`                                            | Additional arguments to otelcol container.                                                                                                                                                                                                                                                                                                                           | `["--config=/etc/otel/config.yaml"]`                                                                                                                                                 |
```

Comment:
The default argument here is confusing imo, as it's the same as the default, but we're adding to the default instead of replacing it. Can we just make the default empty instead?

Reply:
@swiatekm-sumo If I keep it empty then PR checker fails in comparing readme data with actual received values.

Reply:
You need to make it empty in the commented-out section as well.

Reply:
@swiatekm-sumo I have addressed you comment by removing args from comment section. Can you trigger PR checker?